include/lsst/afw/table/Schema.h

```diff
@@ -46,7 +46,7 @@ class Schema {
 public:
 
     // This variable is defined in SchemaImpl, but is replicated here as
-    // a static so that it is available to SWIG.
+    // a static so that it is available to Python.
     static int const VERSION = detail::SchemaImpl::VERSION;
```

Comment:
I guess we don't expose it to Python (maybe we don't need it, but I'll add it just in case; clearly we needed it at one point).  Agree the comment is clearer without "as a static".